README.md

```diff
@@ -26,186 +56,155 @@ Domain-Driven-Design approach.
 
-## DDD Annotations
+## DDD Annotations ✏️
 
 Domain-Driven-Design concepts are identified with annotations.
 
 ### Strategic patterns
 
-#### Bounded Context
+Only `package-info.java` files can be annotated with `@BoundedContext` or `@SharedKernel`. Every
+other concepts must belong to a strategic pattern.
 
 ```java
 @BoundedContext(name = "", description = "")
-```
-
-Set of elements (classes, services, etc) containing only what is needed in a
-specific context.
-
-Only `package-info.java` files can be annotated with `@BoundedContext`. Every
-other concepts must be inside a bounded context.
 
-#### Shared Kernel
-
-```java
 @SharedKernel(name = "", description = "")
 ```
 
-Set of elements (classes, services, etc) containing only what is needed in a specific context.
-A shared kernel is the intersection of two bounded contexts (same DDD concepts used in each one).
-It is often used to describe the relationship between two (or more) teams that share a small but common model.
-It can only contain value objects.
-
-Only `package-info.java` files can be annotated with `@SharedKernel`.
-They work like bounded context do.
-
-#### Nested context
-
 Nested bounded contexts or nested shared kernels are forbidden and prevented by the plugin when processing classes.
 
 ### Tactical patterns
 
-#### Aggregate
-
 ```java
 @Aggregate(name = "", description = "")
-```
 
-Cluster of domain objects that can be treated as a single unit. It has an
-identity and a lifetime.
+@DomainCommand(description = "")
 
-An aggregate contains the domain logic and is aware of framework issues.
+@DomainEvent(description = "")
 
-It is accessed by commands, returns events and deals with value objects.
+@ValueObject(description = "")
+```
 
-Value objects in an aggregate are usually seen as Domain entities. This is the
-only domain model that can be mutable.
+## Documentation generation 🏗️
 
-#### Domain Command
+Once your project is correctly annotated with
+the [DDD Annotations](#DDD_Annotations), use Maven to run the plugin :
 
-```java
-@DomainCommand(description = "")
+```
+mvn clean install
 ```
 
-A command is the entry point to perform a business logic from an aggregate.
+A JSON file will be generated in the `target/candy-doc` directory. The next step is the candy-board downloading. Use the following commands :
 
-It should use primitives only (and could use value objects) and is performed on
-an aggregate.
+```
+npm i @candy-doc/cli
+candy-doc build /target/candy-doc/ubiquitous-language/
+```
 
-#### Domain Event
+Those two lines will download the Candy-doc CLI and download the Candy-doc UI called **Candy-board**.
 
-```java
-@DomainEvent(description = "")
-```
+Now, a `candy-build` repository appeared in your root directory.
 
-Captures the memory of something interesting which affects the domain. It
-usually starts with a verb in the past tense.
+You can display your documentation by opening the `index.html` file in your favorite browser.
```

Comment:
I think we should link this part to the other projects to avoid sync problems between the way it works now and the future of candy-cli / candy-board.